index.ts

```diff
@@ -1460,14 +1711,19 @@ export const convexTest = <Schema extends GenericSchema>(
   // For example `import.meta.glob("./**/*.*s")`
   modules?: Record<string, () => Promise<any>>,
 ): TestConvex<SchemaDefinition<Schema, boolean>> => {
-  const db = new DatabaseFake(schema ?? null);
-  (global as unknown as { Convex: any }).Convex = {
-    syscall: syscallImpl(db),
-    asyncSyscall: asyncSyscallImpl(db),
-    jsSyscall: jsSyscallImpl(db),
-    db,
-    modules: moduleCache(modules),
-  };
+  const rootDb = new DatabaseFake(schema ?? null, "");
```

Comment:
```suggestion
  const rootDb = new DatabaseFake(schema ?? null, /* componentPath */ "");
```
(or `const ROOT_COMPONENT_PATH = ""`)